lavorare-insieme/tool.md

```diff
@@ -53,3 +52,29 @@ utilizzo:
 - Evitiamo le chat private il più possibile, soprattutto per richieste di aiuto o supporto. Altri
   potrebbero avere una risposta migliore, imparare dalla domanda o semplicemente avere voglia di
   scatenare un divertentissimo flame.
+
+## Float
+
+Per tracciare il lavoro svolto e per organizzare il lavoro da svolgere in futuro, usiamo
+[Float](https://nebulab.float.com).
+
+Ci sono degli eventi ricorrenti a cui bisogna prestare attenzione:
+
+- ogni Venerdì, a fine giornata, ti verrà inviata un'email con la pianificazione del lavoro da svolgere per la settimana seguente;
```

Comment:
*venerdì, minuscolo

Reply:
correggo per seguire lo stile di tutta la guida ma di fatto in Italiano si scrive maiuscolo 😛

Reply:
@mtylty calcola no 😄 http://www.accademiadellacrusca.it/it/lingua-italiana/consulenza-linguistica/domande-risposte/alcuni-quesiti-nomi-mesi-giorni-settimana

E non si scrive maiuscolo neanche "italiano" :trollface: